content/tutorials/node/simple-chat.md

```diff
@@ -53,16 +53,25 @@ From the `chat-app` directory, run `npm run build`. This will run the `./build.s
 
 ### **Run the Server**
 
-Open a new terminal window, navigate to the `chat-server` directory and start the application:
+Open a new terminal window, navigate to the `chat-server` directory, setup the fluvio topics and start the application:
 
 ```bash
```

Comment:
for consistency, we should use the same 1 liner as in the client (&& to chain commands)

Reply:
cd chat-server && npm run setup && npm run start